StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentElement+Internal.swift

```diff
@@ -357,7 +358,8 @@ extension EmbeddedPaymentElement {
                 return (.failed(error: error), nil)
             }
         }
-
+        
+        embeddedPaymentMethodsView.isUserInteractionEnabled = false
```

Comment:
I'm thinking we only need to disable interaction after we have waited for the previous update (if any) since that should handle UI enable/disable for the update call.